ckanext/datarequests/auth.py

```diff
@@ -17,13 +17,26 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with CKAN Data Requests Extension. If not, see <http://www.gnu.org/licenses/>.
 
-from ckan.plugins.toolkit import auth_allow_anonymous_access, get_action
+from ckan import authz
+from ckan.plugins.toolkit import asbool, auth_allow_anonymous_access, config, get_action
 
 from . import constants
 
 
 def create_datarequest(context, data_dict):
-    return {'success': True}
+    return {
+        'success': asbool(config.get("ckanext.auth.create_datarequest_if_not_in_organization", "True"))
```

Comment:
@ThrawnCA Leaving this in won't break anything right, would our cookbook need updates to allow datarequests again or was that on a 'fork'?

Reply:
It defaults to True when not provided, which matches the old behaviour, so there should be no problem.